proposals/drafts/elections.rst

```diff
@@ -1,37 +1,249 @@
+
+The typical implementation will be an ``all_day`` event witht a "election" ``classification`` value and a ``start_time`` set to midnight of the observed election date.
+
 id
-    ocd identifier
+    Open Civic Data-style id in the format ``ocd-election/{{uuid}}``.
 
 identifiers
-    upstream identifer
+    **optional**
+    **repeated**
+    Upstream identifiers of the election if any exist, such as those assigned by a Secretary of State, county or city elections office.
+
+administrative_org_id
+    **optional**
+    Reference to the ``Organization`` that administers the election.
+
+state
+    `FIPS code <https://en.wikipedia.org/wiki/Federal_Information_Processing_Standard_state_code>`_ of the state where the election is being held. Recorded in the format ``st{{fips}}`` to match references to VIP `<State> <http://vip-specification.readthedocs.io/en/release/built_rst/xml/elements/state.html>`_  elements (string).
+
+is_statewide
+    **optional**
+    Indicates whether the election is statewide (boolean).
+
+registration_info
+    **optional**
+    Specifies information about registration for this election, either as text or a URI (string).
+
+registration_deadline
+    **optional**
+    Specifies the last day to register for the election with the possible exception of Election Day registration (date).
+
+has_election_day_registration
+    **optional**
+    Specifies if a voter can register on the same day of the election, i.e., the last day of the election (boolean).
+
+absentee_ballot_info
+    **optional**
+    Specifies information about requesting absentee ballots either as text or a URI (string).
+
+absentee_request_deadline
+    **optional**
+    Specifies the last day to request an absentee ballot (date).
+
+results_uri
+    **optional**
+    Contains a URI where results for the election may be found (string).
+
+
+ContestBase
+-----------
+
+A base class with the properties shared by all contest types: ``BallotMeasureContest``, ``CandidateContest``, ``PartyContest`` and ``RetentionContest``.
+
+id
+    Open Civic Data-style id in the format ``ocd-contest/{{uuid}}``.
+
+identifiers
+    **optional**
+    **repeated**
+    Upstream identifiers of the contest if any exist, such as those assigned by a Secretary of State, county or city elections office.
+
+abbreviation
+    **optional**
+    Abbreviation for the contest (string).
+
+ballot_subtitle
+    **optional**
+    Subtitle of the contest as it appears on the ballot (string).
+
+ballot_title
+    **optional**
+    Title of the contest as it appears on the ballot (string).
+
+division_id
+    **optional**
+    Reference to the ``Division`` that defines the geographical scope of the contest, e.g., a specific Congressional or State Senate district.
```

Comment:
Ah, so this seems like it's where you're specifying the thing I was describing with jurisdiction pointers above. Maybe the jurisdiction pointer/is_statewide isn't necessary since this data is found here?